src/main/cpp/t06_sum.cpp

```diff
@@ -21,8 +21,18 @@
 #include <cmath>
 
 using namespace std;
-
+int sum(int i, int summ) {
+	if (i == 0)
+		return summ;
+	else{
+		summ += i;
+		cin >> i;
+		summ = sum(i, summ);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'summ' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4513130)